subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -595,6 +594,8 @@ static void isr_done(void *param)
 	}
 
 #if defined(CONFIG_BT_CTLR_ADV_INDICATION)
+	struct node_rx_hdr *node_rx;
```

Comment:
I think it's better to combine the declaration with the initialization:

    struct node_rx_hdr *node_rx = ull_pdu_rx_alloc_peek(3);

but that'd just be a suggestion.  The last two seem to do that.